### ISSUE TITLE
Custom view window, start-of-week, day-of-week

### ISSUE THREAD

**Issue by alyzz** [fullcalendar/fullcalendar#3113]
I'm using full calendar scheduler for scheduling tasks.

But now, I have a problem: I'm using custom view "timelineFivedays" with duration of 5 days. But the 5 days always start on the current day, I want to start on Monday for the current week to dispaly Monday to Friday. firstDay: 1 doesn't seem to work also. I tried using timelineWeek then hiddenDays = [0,6] to hide the weekends but I get iterating errors in angular instead. 

Can you help me please?

This is how I structured my custom view:

```
                     views: {
                        timelineFivedays: {
                            type: 'timelineWeek',
                            duration: {
                                days: 5
                            }, 
                            intervalStart:1,
                            firstDay: 1,
                            buttonText: '',
                            columnFormat: 'dddd D MMMM',
                            titleFormat: 'dddd D MMMM'
                        }
                    },
                    header: headerTitle,
                    defaultView: timelineFivedays,
```

**Comment by arshaw** [comment]
This is currently not possible. I could see a potential API change like this:

```
views: {
    timelineFiveDays: {
        duration: { days: 5 },
        startOf: 'week',
        dayOfWeek: 1
    }
}
```

or something like that.

somewhat related to https://github.com/fullcalendar/fullcalendar/issues/3105

**Comment by arshaw** [comment]
similar to #3105

**Comment by jeremymcminn** [comment]
In viewRender I used the following code as a workaround, however this disables the next and previous buttons :/

```
var calDateNew = $('#calendar').fullCalendar('getDate');
var calDateNew = moment(calDateNew).startOf('isoWeek');
$('#calendar').fullCalendar('gotoDate', calDateNew);
```

**Comment by arshaw** [comment]
Released in **v3.3.0**
Blog post: https://fullcalendar.io/blog/2017/03/fullcalendar-33-and-scheduler-16/

[dateAlignment](https://fullcalendar.io/docs/current_date/dateAlignment/)
